esphome/components/lcd_base/lcd_display.h

```diff
@@ -23,6 +23,8 @@ class LCDDisplay : public PollingComponent {
   float get_setup_priority() const override;
   void update() override;
   void display();
+  //// Clear LCD display
+  void clear(void);
```

Comment:
See https://travis-ci.org/esphome/esphome/jobs/600625094#L860

```suggestion
  void clear();
```